packages/zoe/package.json

```diff
@@ -47,5 +47,17 @@
     "tap-spec": "^5.0.0",
     "tape": "^4.11.0",
     "tape-promise": "^4.0.0"
-  }
+  },
+  "files": [
```

Comment:
I would prefer not specifying files, and instead using .npmignore.

YMMV.

Reply:
Hmm, if it helps, future changes to Zoe should drastically reduce the number of files listed here. I generally prefer the whitelist approach myself.